README.md

```diff
@@ -1,6 +1,9 @@
 # Bitstyles
 
-A developing collection of CSS styles and helpers, for use in Bitcrowd projects.
+A collection of CSS objects and helpers. See the [Bitcrowd frontend styleguide](https://github.com/bitcrowd/frontend) for details of the principles on which this collection is built. The TL;DR: it’s a variation of the BEMATOMICOOCSS methodology:
```

Comment:
There’s certainly no grammatical rule about this, it’s just a question of style. Personally I’d never do that as it makes the pre-colon text feel like it doesn’t belong… but I don’t care about it so much!

Here is a page with a bunch of arbitrary decisions someone somewhere has made about this, that corroborates my style choice :D http://data.grammarbook.com/blog/commas/capitalization-with-colons/
(they appear to violate their own rules in some places)